builtin/providers/google/resource_compute_vpn_tunnel_test.go

```diff
@@ -32,6 +32,41 @@ func TestAccComputeVpnTunnel_basic(t *testing.T) {
 	})
 }
 
+func TestAccComputeVpnTunnel_noopdiff(t *testing.T) {
+
+	testCheck := func(s *terraform.State) error {
+		return nil
+	}
+
+	testCheckChange := func(s *terraform.State) error {
+		return nil
+	}
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckComputeVpnTunnelDestroy,
+		Steps: []resource.TestStep{
+			resource.TestStep{
+				Config: testAccComputeVpnTunnelNoopChange,
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckComputeVpnTunnelExists(
+						"google_compute_vpn_tunnel.foobar"),
+					testCheck,
+				),
+			},
+			resource.TestStep{
```

Comment:
Does this test fail on master with just one of the test steps? TF tests usually check that running 'plan' again wouldn't cause a diff, so you shouldn't actually need to check the same config twice.